registration/models.py

```diff
@@ -70,6 +70,10 @@ def __str__(self):
     class Meta:
         verbose_name_plural = "students"
 
+    def clean(self):
+        validate_student(self)
+        return super().clean()
```

Comment:
What does `super().clean()` do?

Reply:
it makes sure it calls the OG `clean` method, which will ensure all the other fields are also cleaned 🧼